HealthcareSystem/PatientWebApp/DTOs/SurveyDTO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientWebApp.DTOs
+{
+    public class SurveyDTO
+    {
+        public int Id { get; set; }
+        public int BehaviorOfDoctor { get; set; }
+        public int DoctorProfessionalism { get; set; }
+        public int GettingAdviceByDoctor { get; set; }
+        public int AvailabilityOfDoctor { get; set; }
+        public int BehaviorOfMedicalStaff { get; set; }
+        public int MedicalStaffProfessionalism { get; set; }
+        public int GettingAdviceByMedicalStaff { get; set; }
+        public int EaseInObtainingFollowupInformationAndCare { get; set; }
+        public int Nursing { get; set; }
+        public int Cleanliness { get; set; }
+        public int OverallRating { get; set; }
+        public int SatisfiedWithDrugAndInstrument { get; set; }
+        public string DoctorJmbg { get; set; }
+        public string DoctorName { get; set; }
+        public string DoctorSurname { get; set; }
+
+        public SurveyDTO() { }
+
+    }
```

Comment:
I am not sure about this, is good to have too many fields in DTO like this? Maybe we should make some structure for some of the properties? What do you think about that @jelenazeljko55 ?

Reply:
It's a DTO though, there might be some advantages to keeping it flat. I think we might learn more about that when we get to Domain Driven Design?

Reply:
I am not sure about that, can you give me arguments about why is a better solution to have a 'flat' dto?

Reply:
But if we will learn more about that in DDD, we can keep this or ask some of the product owners.

Reply:
I'm honestly not sure, looking up the best practices for DTO structure pretty much only gives me "it depends".  My logic was mostly that since their purpose is just getting data from one place to another, the code that uses them might be cleaner if it doesn't have to always string together a bunch of getters to reach the data it needs. For this case my personal taste would probably be to maybe pull out a DoctorDTO as you suggested, since that can be reused in other DTOs as a foreign key representation of the Doctor class. But I don't really like the GradesDTO suggestion, the ratings are only relevant to the survey so leaving them there directly makes sense to me. 

But really at this point I would be alright with any approach to the DTO structure that @jelenazeljko55 chooses, since it's mostly a matter of personal preference. The only thing I would insist on is keeping it consistent within the PatientWebApp. From that aspect it might be good to note that the other DTOs in the PatientWebApp are flat.

Reply:
@jelenazeljko55 feel free to inform us what you decide

Reply:
I would change DTO class in the future (on some other branch)... is that OK?

I agree with is splitting up the ValidateSurveyFields method...

Reply:
That is okay for me.